docs/demos/1-simple/index.html

```diff
@@ -26,7 +26,7 @@ <h1>Simple</h1>
             <thead>
                 <tr>
                     <th>
-                        <b>N</b>ame
+                        <b>Name</b>
```

Comment:
What is the need for this change here? Are there issues when only part of the contents of a cell are styled?

Reply:
Just an artifact of my testing; I'll revert it.

Was checking if my eyes were playing me a trick, but it seems that the headers are already bold with the default styling. Maybe a better option would be to have this header in italics, so it is visually distinct from the others?